spec/factories/trainees.rb

```diff
@@ -64,8 +64,31 @@
       outcome_date { Faker::Date.in_date_period }
     end
 
-    trait :trn_requested do
-      trn_requested_at { Time.zone.now }
+    trait :submitted_for_trn do
+      state { "submitted_for_trn" }
+      submitted_for_trn_at { Time.zone.now }
```

Comment:
Should that be a little bit more randomised?

Reply:
If a test needs to specify it can be passed in. Why does it need to be random?

Reply:
It getting used here...
https://github.com/DFE-Digital/register-trainee-teachers/blob/a29acb08b524e8dae559c379c82b9647cf0be16d/lib/tasks/example_data.rake#L25-L26

do we want to have demo data that singular timed line ?

Reply:
Ok. I've left the factory but I've tried to make the dates a bit more random and hopefully still sensible in the rake task. Also added the TRN when it should be there.